xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.h

```diff
@@ -118,7 +118,7 @@ namespace Shaders {
     GLint m_hYTex;
     GLint m_hUTex;
     GLint m_hVTex;
-    GLint m_hMatrix;
+    GLint m_hMatrix = -1;
```

Comment:
do you know the meaning of a program handle for shaders? if not, i suggest to leave as is